py_proxy/views.py

```diff
@@ -43,7 +44,7 @@ def pdf(request):
     return {
         "pdf_url": f"{nginx_server}/proxy/static/{pdf_url}",
         "client_embed_url": request.registry.settings["client_embed_url"],
-        "h_open_sidebar": int(request.params.get("via.open_sidebar", "0")),
+        "h_open_sidebar": asbool(request.params.get("via.open_sidebar", False)),
```

Comment:
Can we also, as part of this PR, change the default value from `False` to `True`? Currently if I go to https://via3.hypothes.is/ and paste in a PDF URL it'll proxy the PDF but not open the sidebar. While this is the same behaviour as legacy Via, it almost certainly isn't what users want (the point of proxying things through Via is supposed to be to annotate them or to view annotations).

Alternatively we could remove `via.open_sidebar` as an option entirely and just always pass `openSidebar: true` to the client. I'm not sure there's ever really a reason to proxy something through Via and _not_ open the sidebar.

Leaving the option in but just defaulting it to `True` is also fine, though

Reply:
Opened a separate issue for this: https://github.com/hypothesis/py-proxy/issues/39